tests/python/test_ad_basics.py

```diff
@@ -76,6 +76,7 @@ def test_poly():
 
 
 @if_has_autograd
+@ti.test(exclude=[ti.vulkan])
 def test_trigonometric():
     grad_test(lambda x: ti.tanh(x), lambda x: np.tanh(x))
     grad_test(lambda x: ti.sin(x), lambda x: np.sin(x))
```

Comment:
`ti.vulkan` seems not support autograd yet...

Reply:
This seems to be running fine on vulkan, except for an accuracy problem too :P 
`FAILED tests/python/test_ad_basics.py::test_trigonometric - assert 1.3344652652740479 == 1.3346064424467945 ± 1...`

Reply:
But it's also need to note that the accuracy in Vulkan backend seems lower than other backends...